### DISCUSSION TITLE
Yet another idea on modeling DRAM power usage

### DISCUSSION THREAD

**Discussion by rootfs** [sustainable-computing-io/kepler#1178] [Ideas]
Would [DAMON tracepoints](https://docs.kernel.org/next/admin-guide/mm/damon/usage.html#tracepoints-for-monitoring-results) be useful in modeling DRAM power? It has memory access pattern stats in it so maybe we can use it for power modeling.

@marceloamaral @sunya-ch

**Comment by marceloamaral** [comment]
@rootfs this is interesting.

Currently, we are measuring the virtual page cache access.
With a quick view, it seems that DAMON is also doing that.
https://docs.kernel.org/next/mm/damon/design.html#access-frequency-monitoring

But it might be worth checking if the metric is different.
